application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1801,6 +1801,10 @@ type RealTimeEstimate {
 type RentalVehicle implements Node & PlaceInterface {
   "If true, vehicle is currently available for renting."
   allowPickupNow: Boolean
+  "Fuel or battery power remaining in the vehicle. Expressed from 0 to 1."
+  currentFuelPercent: Float
```

Comment:
We have a scalar called Ratio, I'm not sure if that would be a good fit for this as well. I also have some intuition that these two new fields should be grouped together under some object but I don't know what to call it nor am I sure if it's actually a good idea.

Reply:
Ratio seems a good fit